scss/_carousel.scss

```diff
@@ -355,7 +356,7 @@
   left: (100% - $carousel-caption-width) * .5;
   padding: $carousel-caption-padding-y $carousel-caption-padding-x; // Boosted mod
   color: $carousel-caption-color;
-  background-color: $carousel-caption-bg; // Boosted mod
+  background-color: $carousel-caption-bg; // Boosted mod: instead of `var(--#{$prefix}carousel-caption-color)`
```

Comment:
[On the bootstrap side](https://github.com/twbs/bootstrap/commit/0cbfe13adf669ad39ae9d8e873c2ad59befd3a3a#diff-520d0371d3285ebe93f02dc9b5c5d8aa5dc4edb489a1ab55947e2348f0cc5a4dL198) the change was on the color, not the background color but there is no background color on Bootstrap side. Shouldn't we update the color instead ?

Reply:
Good catch ! I've changed the comment so we aren't lost. Since we already properly take care of the dark mode carousel in Boosted, I didn't change anything.